diff --git a/base/BUILD.gn b/base/BUILD.gn
index 2d866b203adf1dfd81dbc4b22b02356c59b41133..db71cb06ddf10d7e7c8bc93f7d7288b0a7ef706f 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4943,6 +4943,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5610,6 +5611,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
