diff --git a/chrome/browser/ui/views/toolbar/toolbar_view.cc b/chrome/browser/ui/views/toolbar/toolbar_view.cc
index 7f352cd1c83aa09de95a2a8358a6cdd12230b788..2fc46b8b84512de77405b8ec62b6532be8c0ae34 100644
--- a/chrome/browser/ui/views/toolbar/toolbar_view.cc
+++ b/chrome/browser/ui/views/toolbar/toolbar_view.cc
@@ -571,6 +571,7 @@ void ToolbarView::Init() {
     }
   }
 
+  BRAVE_TOOLBAR_VIEW_INIT
   avatar_ = AddChildView(std::make_unique<AvatarToolbarButton>(browser_view_));
   bool show_avatar_toolbar_button = true;
 #if BUILDFLAG(IS_CHROMEOS)
