diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 1424bbeb94eaa1f9952d93b39fab7d79531cdecf..580b6600f3d84e70ee6f9cacce31fcc952edb6b6 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4584,6 +4584,7 @@ const FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
