diff --git a/chrome/browser/browsing_data/BUILD.gn b/chrome/browser/browsing_data/BUILD.gn
index 50ffc5a0db48e51fdebf3bc5d125985bdb02ae35..37dbbc5934dbc23113d2c0739c5521e9adc04986 100644
--- a/chrome/browser/browsing_data/BUILD.gn
+++ b/chrome/browser/browsing_data/BUILD.gn
@@ -12,6 +12,7 @@ source_set("constants") {
     "//content/public/browser",
     "//extensions/buildflags",
   ]
+  public_deps = [ "//brave/components/ai_chat/core/common/buildflags" ]
 }
 
 source_set("browsing_data") {
