diff --git a/chrome/android/java/src/org/chromium/chrome/browser/multiwindow/MultiInstanceOrchestratorImpl.java b/chrome/android/java/src/org/chromium/chrome/browser/multiwindow/MultiInstanceOrchestratorImpl.java
index bf1e2d0dac1a24b9d58a8fa574d94e9d51e17aa3..5d8c575c075125a2879e6760fac8e2102c91c9c8 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/multiwindow/MultiInstanceOrchestratorImpl.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/multiwindow/MultiInstanceOrchestratorImpl.java
@@ -42,7 +42,7 @@ import java.util.Set;
 
 /** Implements {@link MultiInstanceOrchestrator} as a singleton. */
 @NullMarked
-/* package */ final class MultiInstanceOrchestratorImpl implements MultiInstanceOrchestrator {
+/* package */ class MultiInstanceOrchestratorImpl implements MultiInstanceOrchestrator {
     private static @Nullable MultiInstanceOrchestrator sInstance;
     private static @Nullable TabReparentingDelegate sTabReparentingDelegateForTesting;
 
@@ -63,7 +63,7 @@ import java.util.Set;
         return sInstance;
     }
 
-    private MultiInstanceOrchestratorImpl(TabReparentingDelegate tabReparentingDelegate) {
+    MultiInstanceOrchestratorImpl(TabReparentingDelegate tabReparentingDelegate) {
         mTabReparentingDelegate = tabReparentingDelegate;
         ApplicationStatus.registerStateListenerForAllActivities(this::onActivityStateChange);
     }
