diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index f25ef01b67ad01e15fd0310cbdaf723d741e2a55..16ded2dd7e9515a918d634d65d533129e7ea3847 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -1542,6 +1542,7 @@ source_set("extensions") {
       "api/tabs/app_window_helper.h",
     ]
   }
+  deps += brave_extensions_resources
 }
 
 static_library("test_support") {
