diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index ee6ff0cb455bf79c08a9575b6e217d2552cce2d3..3d8e58a5596d18ae5d5075c75391c735d4d09e1d 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2360,6 +2360,7 @@ static_library("browser") {
   } else {
     sources += [ "net/net_error_diagnostics_dialog_stub.cc" ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (is_android) {
     deps += [
@@ -7753,6 +7754,7 @@ group("browser_public_dependencies") {
       "//components/offline_pages/core/background:background_offliner",
     ]
   }
+  import("//brave/browser/sources.gni") public_deps += brave_chrome_browser_public_deps
 }
 
 # All generated files in //chrome/browser/ depended on by
