diff --git a/chrome/browser/ui/chrome_pages.cc b/chrome/browser/ui/chrome_pages.cc
index 3f0cc53959f0d6ceb9bc57ce5f1c2cc2b38866b8..225d212e81c590284768b772c727fb333ff39a97 100644
--- a/chrome/browser/ui/chrome_pages.cc
+++ b/chrome/browser/ui/chrome_pages.cc
@@ -426,7 +426,7 @@ void ShowSlow(BrowserWindowInterface* browser) {
 #endif
 }
 
-GURL GetSettingsUrl(std::string_view sub_page) {
+BRAVE_GET_SETTINGS_URL
   return GURL(base::StrCat({kChromeUISettingsURL, sub_page}));
 }
 
