diff --git a/chrome/android/chrome_java_sources.gni b/chrome/android/chrome_java_sources.gni
index 6d8fa44228391fa91047bed64784532ee073e883..1e8609ad7c9d2f99465ed5dea6207d123e431042 100644
--- a/chrome/android/chrome_java_sources.gni
+++ b/chrome/android/chrome_java_sources.gni
@@ -1340,3 +1340,4 @@ if (enable_screen_capture) {
     "java/src/org/chromium/chrome/browser/media/MediaCapturePickerTabObserver.java",
   ]
 }
+chrome_java_sources -= ["java/src/org/chromium/chrome/browser/ntp/IncognitoNewTabPageView.java",]
