diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 8278adcb0a3067889091363178117672888fcec3..622620ee8eb954213f4d7d42d4d8bd0d881990ea 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3776,6 +3776,7 @@ source_set("browser") {
       "surface_embed/surface_embed_connector_impl.h",
     ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
