diff --git a/ios/chrome/browser/flags/about_flags.mm b/ios/chrome/browser/flags/about_flags.mm
index 4b2bd325d47a1459bdd2656dfb350ae4c0a0e85c..2421060df29d0ccce36bd3dce4297391d32acbb3 100644
--- a/ios/chrome/browser/flags/about_flags.mm
+++ b/ios/chrome/browser/flags/about_flags.mm
@@ -1378,6 +1378,7 @@ const FeatureEntry::FeatureVariation kAssistantContainerVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 constexpr auto kFeatureEntries = std::to_array<flags_ui::FeatureEntry>({
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
     {"build-external-privacy-context",
      flag_descriptions::kBuildExternalPrivacyContextName,
      flag_descriptions::kBuildExternalPrivacyContextDescription,
