diff --git a/BUILD.gn b/BUILD.gn
index fc8e4b4cf665de72150607a60f33abb6b15cfc18..33172c179b368e777934397887b8ac0fe2fc5225 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -900,6 +900,7 @@ config("internal_config_base") {
     "$target_gen_dir",
     "$target_gen_dir/include",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3545,6 +3546,7 @@ v8_header_set("v8_headers") {
     ":v8_tracing",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
